### PR TITLE
Docs/html doc issues

### LINKED ISSUE #1005: New html docs issue
1. contents layout is weird:
Screenshot 2023-12-06 at 9 41 15 AM
2. could we remove this notification? `stable` is a very old tag:
Screenshot 2023-12-06 at 9 42 14 AM
3. there should be an interactive plot that shows the curve, but is missing on newest docs, same as atari:
Screenshot 2023-12-06 at 1 35 08 PM
Screenshot 2023-12-06 at 1 36 03 PM
4. contributor page is old
Screenshot 2023-12-06 at 1 37 28 PM

### PR DESCRIPTION
**Pull request by MischaPanch**
Closes #1005 

## Main changes

1. Removes the failing softlinks, replaced by copy-pasted results.json
2. Load vega-embed things using jupyter-book config 
3. Add vega-embed dependencies as part of local code for offline development
4. Reduced duplication in benchmark.js

The results look really nice, we get additional features from jupyter-book and vega

image

### CODE REVIEW

**Comment by MischaPanch** [comment]
@carlocagnetta 

When I build locally, even with new dependencies, I still have a problem with the contents/references 

image

**Comment by carlocagnetta** [comment]
@MischaPanch My guess is that it was working locally on my machine bc I have docutils==20.0.0 installed. Thing is, pyproject did not allow me to update the min required version because of other dependencies requiring <20.0

**Comment by MischaPanch** [comment]
Ok, then we can't fix it until jupyter-book bumps docutils. Is there an issue at their repo for that already?

**Comment by MischaPanch** [comment]
Ok, updated sphinx and docutils, removed jquery. Now references look nice again and benchmarks widgets work. Ready to merge from my side

File: pyproject.toml [hunk 1]
```diff
@@ -45,7 +45,11 @@ virtualenv = [
     { version = "<20.16.4", markers = "sys_platform == 'win32'" },
 ]
 
-# TODO: add versions
+
+# Atari, box2d, classic-control, and mujoco environments are all optional dependencies of gymnasium.
+# Unfortunately, we cannot have extras relying on (multiple) optionals of other packages due to a poetry issue (see e.g.
+# https://github.com/python-poetry/poetry/issues/7911) and therefore have to maintain our own list of dependencies.
+# This requires attention and monitoring of gymnasium's dependencies and their version numbers!
```
Comment (carlocagnetta) [inline pyproject.toml:52]: This moved up here, maybe we should move it back down to the extras.

**Review by carlocagnetta — COMMENTED** [review]
Wonderful, this should close all current docs issues.
We can continue working on re-wording and spelling in future minor PR, but this should work for the release.

File: pyproject.toml (hunk 1 above)
Comment (MischaPanch) [reply pyproject.toml:52]: Already fixed

Comment (MischaPanch) [reply pyproject.toml:52]: Ah, in fact it's still moved, but this is an even better place for the comment

**Review by Trinkle23897 — DISMISSED** [review]

**Comment by carlocagnetta** [comment]
image
Unfortunately it does not seem to be resolved. Still problems building in ReadTheDocs

**Pull request merged**